include/seqan3/utility/char_operations/predicate.hpp

```diff
@@ -55,16 +54,13 @@ inline constexpr auto is_in_interval = detail::is_in_interval_type<interval_firs
  *
  * \details
  *
- * This function like object returns true for all characters of the alphabet, false otherwise.
- * The actual check being performed is whether assigning and then reading a letter results in the original input
- * (but case is ignored).
+ * This function is deprecated and returns always true.
  *
- * ### Example
- * \include test/snippet/utility/char_operations/char_predicate_is_in_alphabet.cpp
  * \hideinitializer
+ * \deprecated Use seqan3::char_is_valid_for instead.
  */
-template <alphabet alphabet_t>
-inline constexpr auto is_in_alphabet = detail::is_in_alphabet_type<alphabet_t>{};
+template <typename alphabet_t>
+SEQAN3_DEPRECATED_310 inline constexpr bool is_in_alphabet = true;
 
 /*!\brief Checks whether a given letter is the same as the template non-type argument.
```

Comment:
Should that not be deleted?

Reply:
In the issue it states **deprecate** seqan3::is_in_alphabet, so I did not delete it.